### PR TITLE
Additional helpers

### PR DESCRIPTION
**Pull request by smondet**
Mostly about creating Ketrew hosts and Biokepi machines automaticatlly.

### CODE REVIEW

File: src/lib/deploy.ml [hunk 1]
```diff
@@ -1109,6 +1136,65 @@ module Cluster = struct
...
+      ?(gatk_jar_location = `Fail "Cannot use GATK without the JAR location")
+      ?(mutect_jar_location = `Fail "Cannot use Mutect without the JAR location")
+      t ~configuration ~work_dir =
+    let host = ketrew_host t ~work_dir in
+    let max_processors =
+      match t.torque_server.Node.machine_type with
+      | `Google_cloud g ->
+        begin match g with
+        | `Small -> 1
+        | `Highmem_8 -> 8
```
Comment (ihodes) [inline src/lib/deploy.ml]: is that 8gb == high? That seems lower than the defaults we have in Hamkepi?

**Comment by ihodes** [comment]
LGTM, 1 q

File: src/lib/deploy.ml (hunk 1 above)
Comment (smondet) [inline src/lib/deploy.ml]: 8 if the number of processors. 
The specs are: “`n1-highmem-8 (8 vCPUs, 52 GB memory)`”
I use these for now, I plan to run more benchmarking to see what we really need.
Also for now the PBS cluster is configured to be homogeneous; it has some "self-config" capabilities; but I don't know yet how to do a proper heterogeneous cluster.

Comment (ihodes) [inline src/lib/deploy.ml]: Ah, cool. Mind adding a comment on what these mean?

**Pull request merged**